<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diode_dlg</class>
 <widget class="QDialog" name="diode_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diode Properties</string>
  </property>
  <widget class="QLineEdit" name="box_diode_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Diode Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_rev_current">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rev_current">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current - Reverse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_voltage_fwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fwd_voltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage - Forward</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_power_max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_power_max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Power - Max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_capacitance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_capacitance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacitance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_impedance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_impedance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Impedance - Max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_zener_voltage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_zener_voltage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage - Zener</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_reverse_recovery_time">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_rrt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse Recovery Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>box_diode_type</tabstop>
  <tabstop>box_rev_current</tabstop>
  <tabstop>box_voltage_fwd</tabstop>
  <tabstop>box_power_max</tabstop>
  <tabstop>box_tolerance</tabstop>
  <tabstop>box_capacitance</tabstop>
  <tabstop>box_impedance</tabstop>
  <tabstop>box_zener_voltage</tabstop>
  <tabstop>box_reverse_recovery_time</tabstop>
  <tabstop>btn_OK</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
